src/tests/build.sh

```diff
@@ -176,7 +176,9 @@ __RepoRootDir="$(cd "$__ProjectRoot"/../..; pwd -P)"
 __TargetArch=
 
 handle_arguments_local() {
-    case "$1" in
+    opt="$(echo "${1/#--/-}" | tr "[:upper:]" "[:lower:]")"
```

Comment:
Please also replace other five `$1` with `$opt`. At some point we would make it use common handler in `eng/native/build-commons.sh` and integrate it with build.sh, so we won't need to update these internal scripts.

Reply:
Aren't those $1 uses extracting paths? We don't want to case-normalize paths.

Reply:
They all look like paths to me except for the UnprocessedBuildArgs one, which I think probably also should be treated as case-sensitive since it could be a msbuild property containing a path.

Reply:
True, and the unprocessed ones are passed down to msbuild. Looks good as is. 👍